packages/openid4vc/src/openid4vc-issuer/OpenId4VcIssuerService.ts

```diff
@@ -998,17 +1004,19 @@ export class OpenId4VcIssuerService {
       'pre-authorized_grant_anonymous_access_supported': true,
 
       jwks_uri: joinUriParts(issuerUrl, [config.jwksEndpointPath]),
+
       authorization_challenge_endpoint: joinUriParts(issuerUrl, [config.authorizationChallengeEndpointPath]),
+      authorization_endpoint: joinUriParts(issuerUrl, [config.authorizationEndpoint]),
 
-      // TODO: PAR (maybe not needed as we only use this auth server for presentation during issuance)
-      // pushed_authorization_request_endpoint: '',
-      // require_pushed_authorization_requests: true
+      pushed_authorization_request_endpoint: joinUriParts(issuerUrl, [config.pushedAuthorizationRequestEndpoint]),
+      require_pushed_authorization_requests: true,
 
       code_challenge_methods_supported: [PkceCodeChallengeMethod.S256],
       dpop_signing_alg_values_supported: issuerRecord.dpopSigningAlgValuesSupported,
     } satisfies AuthorizationServerMetadata
 
     return {
+      originalDraftVersion: Openid4vciDraftVersion.V1,
```

Comment:
Thinking, how do we handle the multi-draft issuer metadata, because they are incompatible right? Especially the credential configurations.

Reply:
I think we should add an option when calling createIssuer/uodateIssuer to specify the version? That's how i'd handle this in Pararym i think.

Can be a separate PR